### ISSUE TITLE
Concat operation wrong dim?

### ISSUE THREAD

**Issue by oliverzzm** [namedBen/Convolutional-Pose-Machines-Pytorch#10]
Hi, I read your cpm_model.py and found you assign dim = 1 when doing torch.cat.
But the tensor in pytroch is in c,h,w. So shouldn't dim be 0?
